documentation/ODEs and Runge-Kutta integrators.tex

```diff
@@ -271,7 +271,7 @@ \section*{Definition}
        &  b_{1}  &  \cdots &  b_{s}
 \end{array}
 \]
-We will only consider \emph{explicit} Runge-Kutta methods, \emph{i.e.}, those where $\matA$ is strictly lower triangular, so that the Butcher tableau is as follows (blank spaces in $\matA$ are zeros).\[
+We will only consider \emph{explicit} Runge-Kutta methods, \emph{\idest}, those where $\matA$ is strictly lower triangular, so that the Butcher tableau is as follows (blank spaces in $\matA$ are zeros).\[
```

Comment:
Why the `\emph`?

Reply:
ah yes, that is too many emphs (and they cancel each other out, too).